tests/test_cli_interactive.py

```
@@ -0,0 +1,222 @@
+        "TV",  # Symbol
+        "absolute",  # Fee Type
+        "10",  # Perf Fee
+        "1",  # Mgmt Fee
+        "0x0",  # DAC
+        "Exit",  # Main menu loop again
+    ]
+
+    # We need a shared iterator for the side effect
+    iterator = iter(ask_side_effect)
+
+    def next_answer():
```

Ruff (0.14.11)

Missing return type annotation for private function `next_answer`

(ANN202)